codeclimate-wrapper

```diff
@@ -41,17 +41,34 @@ else
     && invalid_docker_host "$DOCKER_HOST"
 fi
 
+CC_XDG_CONFIG_HOME=${XDG_CONFIG_HOME:-~/.config/}
+CC_XDG_CONFIG="${CC_XDG_CONFIG_HOME}/codeclimate/config.yml"
+CC_XDG_CONFIG_DIR=$(dirname "${CC_XDG_CONFIG}")
+[ -d "$CC_XDG_CONFIG_DIR" ] || mkdir -p "$CC_XDG_CONFIG_DIR"
+[ -f "$CC_XDG_CONFIG" ] || ( echo "---" > "$CC_XDG_CONFIG" )
+
+CC_XDG_CACHE_HOME=${XDG_CACHE_HOME:-~/.cache/}
+CC_XDG_CACHE="${CC_XDG_CACHE_HOME}/codeclimate/cache.yml"
+CC_XDG_CACHE_DIR=$(dirname "${CC_XDG_CACHE}")
+[ -d "$CC_XDG_CACHE_DIR" ] || mkdir -p "$CC_XDG_CACHE_DIR"
+[ -f "$CC_XDG_CACHE" ] || ( echo "---" > "$CC_XDG_CACHE" )
+
 docker_run() {
-  exec docker run \
+  docker run \
```

Comment:
It's a leftover of an intentional change. I started with a directory having only XDG files in it that was mounted into a container and used `trap EXIT` to clean up that dir. With exec script process got replaced by `exec` and there was no one to actually clean up.

I can change it back.

Is there any particular reason why we want docker to become the main process?